### DISCUSSION TITLE
社区版后台 `系统->应用市场->许可证管理` 是空的，社区版能否不显示这个 `许可证管理` 入口

### DISCUSSION THREAD

**Discussion by mx2913** [halo-dev/halo#5847] [General]
### 你当前使用的版本

2.15.0

### 社区版能否不显示这个 `许可证管理` 入口，里面内容是空的

![图片](https://github.com/halo-dev/halo/assets/15864713/65e3e74f-dc64-4414-b069-89d2de32ffc8)

**Comment by ruibaby** [comment]
感谢反馈此问题，但这个是属于应用市场插件的功能，是用于管理付费应用的许可证，如果你完全不需要，可以直接卸载应用市场插件。不过如果是因为这个页面无数据看起来不好看的话，或许可以优化一下。

由于这个 issue 无法直接在当前仓库解决，我将转移到 Discussion。
